src/content/docs/en/pages/agreements/tos/terms-of-service.mdx

```diff
@@ -50,7 +50,7 @@ All Azion products and services are offered as Edge Services and available throu
 
-**Real-Time Events:** designed to enable Customers to analyze Azion's Product and Services' events via web interface, searching events in different Data Sources, with different time filters. AZION WILL USE ALL COMMERCIALLY REASONABLE EFFORTS TO RETAIN THE EVENTS FOR 7 DAYS AND MAKE THEM AVAILABLE TO OUR CUSTOMERS WITH A DELAY LOWER THAN 3 MINUTES. IN CASE OF PEAK ACCESS, ATTACKS OR EXTERNAL FACTORS, THE DELAY MAY BE SUPERIOR THAN THE FORESEEN. Except as otherwise provided for in the Customer Agreement or any other documents attached thereto, Real-Time Events will be billed considering (a) Data Gathered; (b) Real-Time Events Queries, according to the volumes consumed.
+**Real-Time Events**: designed to allow Customers to analyze events generated by: (a) traffic and logs of applications configured on the Azion platform; (b) security configurations; (c) Data Stream integrations and (d) DNS zones hosted on Edge DNS. The queries are customized in GraphQL, enabling search in different Data Sources and application of time filters. Users can monitor and analyze logs in real time via CLI, GraphQL API or web interface, obtaining instant visibility.  AZION WILL USE ALL COMMERCIALLY REASONABLE EFFORTS TO RETAIN THE EVENTS FOR 7 DAYS AND MAKE THEM AVAILABLE TO OUR CUSTOMERS WITH A DELAY LOWER THAN 3 MINUTES. IN CASE OF PEAK ACCESS, ATTACKS OR EXTERNAL FACTORS, THE DELAY MAY BE SUPERIOR THAN THE FORESEEN. Except as otherwise provided for in the Customer Agreement or any other documents attached thereto, Real-Time Events will be billed considering (a) Data Ingested, (b) Storage, and (c) Analyzed Data, according to the volumes consumed.
```

Comment:
```suggestion
**Real-Time Events:** designed to allow Customers to analyze events generated by: (a) traffic and logs of applications configured on the Azion platform; (b) security configurations; (c) Data Stream integrations and (d) DNS zones hosted on Edge DNS. The queries are customized in GraphQL, enabling search in different Data Sources and application of time filters. Users can monitor and analyze logs in real time via CLI, GraphQL API or web interface, obtaining instant visibility.  AZION WILL USE ALL COMMERCIALLY REASONABLE EFFORTS TO RETAIN THE EVENTS FOR 7 DAYS AND MAKE THEM AVAILABLE TO OUR CUSTOMERS WITH A DELAY LOWER THAN 3 MINUTES. IN CASE OF PEAK ACCESS, ATTACKS OR EXTERNAL FACTORS, THE DELAY MAY BE SUPERIOR THAN THE FORESEEN. Except as otherwise provided for in the Customer Agreement or any other documents attached thereto, Real-Time Events will be billed considering (a) Data Ingested, (b) Storage, and (c) Analyzed Data, according to the volumes consumed.
```